Sources/ScipioKit/Runner.swift

```diff
@@ -162,14 +166,16 @@ extension Runner {
                 isDebugSymbolsEmbedded: Bool? = nil,
                 frameworkType: FrameworkType? = nil,
                 extraFlags: ExtraFlags? = nil,
-                extraBuildParameters: [String: String]? = nil
+                extraBuildParameters: [String: String]? = nil,
+                enableLibraryEvolution: Bool? = false
```

Comment:
https://github.com/giginet/Scipio/blob/980d9cebc4276f3ea9bd6aaa7c6f89179b4397ac/Sources/ScipioKit/Runner.swift#L140

On the other hand, the similar default value is true, please unify or add comment about it to make clear

Reply:
Nice catch. it should be `nil`.

```suggestion
                enableLibraryEvolution: Bool? = nil
```